glean-core/android/src/main/java/mozilla/telemetry/glean/rust/LibGleanFFI.kt

```diff
@@ -46,7 +46,9 @@ internal interface LibGleanFFI : Library {
         private val JNA_LIBRARY_NAME = "glean_ffi"
 
         internal var INSTANCE: LibGleanFFI = try {
-            Native.loadLibrary(JNA_LIBRARY_NAME, LibGleanFFI::class.java) as LibGleanFFI
+            val lib = Native.loadLibrary(JNA_LIBRARY_NAME, LibGleanFFI::class.java) as LibGleanFFI
```

Comment:
Ah, I thought there was a Rust specific way to do that in glean-core, rather than in the Kotlin side. No way I guess, @badboy ?

Reply:
No better way than calling `enable_logging` as the first thing in every function. Has the overhead of the constant internal check if it is enabled.